docs/reference/api/drivers/SPI.md

```diff
@@ -23,6 +23,10 @@ Mode |  Polarity |  Phase
 
 The SPI master generates a clock to synchronously drive a serial bit stream slave. The slave returns a bit stream, also synchronous to the clock.
 
+To communicate with multiple slave devices connected over the same SPI peripheral, you must use multiple SPI objects, one for each slave, but instantiate them with different slave selects (SSEL) and configurations corresponding to each slave.
+
+Note that the SPI object automatically configures the SPI peripheral with the right configuration when the application code invokes the interfaces on the SPI object.
```

Comment:
Sorry I'm still spinning up my brain from being out last week. I'm not quite clear on what you mean by 
>...application code invokes the interfaces on the SPI object.
Does this mean when I call the SPI object constructor the SPI hardware will be configured with the provided parameters based on the target automatically?

Reply:
Yes, the SPI constructor calls spi_init() with the provided configuration.

Reply:
Awesome, just making sure. Maybe something like the following would make it a bit more clear? Took me a reread to get what it was going after initially. 

> Note that the SPI object automatically configures the hardware peripheral with the right configuration when the application code invokes the SPI object constructor. 

Or I'm just derping and it's clear as is, I can go either way on this one.